src/components/NewRuntimeModal.js

```diff
@@ -770,6 +775,8 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
                   isClearable: false,
                   onlyInteger: true,
                   value: numberOfWorkers,
+                  disabled: !this.canUpdateNumberOfWorkers(),
+                  tooltip: !this.canUpdateNumberOfWorkers() ? 'Runtime must be in Running status to change number of workers' : undefined,
```

Comment:
I just had a thought - is 'Runtime' the user facing language we use or should it be either 'Cloud Environment' or 'Cloud Compute' maybe?

Reply:
```suggestion
                  tooltip: !this.canUpdateNumberOfWorkers() ? 'Cloud Compute must be in Running status to change number of workers' : undefined,
```

Reply:
will update!